src/main/scala/chisel3/util/Counter.scala

```diff
@@ -25,16 +25,17 @@ import chisel3.internal.naming.chiselName  // can't use chisel3_ version because
   */
 @chiselName
 class Counter(val n: Int) {
-  require(n >= 0)
+  require(n >= 0, "Counter must count to value greater than 0!")
```

Comment:
This text doesn't match the require. The require says "greater than or equal to", but the text just says "greater than"

Reply:
Good point, I misread the require, will fix